next/schema-generator/generator/functions.ts

```diff
@@ -73,7 +73,12 @@ export const select = (
     }),
     uiSchema: () => ({
       'ui:widget': 'Select',
-      'ui:options': uiOptions,
+      'ui:options': {
+        ...uiOptions,
+        selectOptions: options.options
+          .filter(({ description }) => description)
```

Comment:
I don't understand why this filter qualifies it for selectOptions - maybe a comment explaining ? (same below)

if it's something obvious and used like this in many other places just let me know

Reply:
Maybe it makes sense to leave all the options there, but practically it is `uiOptions` and it is only there to extract the description for the options that have it. See https://github.com/bratislava/konto.bratislava.sk/blob/f10c8f0619dbd466587161b279d6d84a750e4c66/next/components/forms/widget-wrappers/SelectWidgetRJSF.tsx#L82